lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -92,6 +93,7 @@ class MailboxDashBoardController extends ReloadableController {
   final DownloadController downloadController = Get.find<DownloadController>();
   final NetworkConnectionController networkConnectionController = Get.find<NetworkConnectionController>();
   final AppGridDashboardController appGridDashboardController = Get.find<AppGridDashboardController>();
+  final MailboxController mailBoxController = Get.find<MailboxController>();
```

Comment:
IMO, it's not advisable to do so. `DashboardController` is an intermediate object that listens for events from other controllers.